npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
It looks like that file is listed in the .gitignore. Not sure why it is showing in the changes. Any ideas?